cmsis/TARGET_CORTEX_M/mbed_tz_context.c

```diff
@@ -22,6 +22,8 @@
  * limitations under the License.
  */
 
+#if !defined(TARGET_TFM)
```

Comment:
I think using the below would be more flexible (it also allows the builder to set the macro to 0/1 in addition to just being set).

```suggestion
#if !TARGET_TFM
```

@deepikabhavnani there was a PR about this recently, can you double check me on this?

Reply:
Thanks for catching this @bridadan. Yes we had a PR about this recently

Reply:
@bridadan `TFM` will be added to the `RAM_MUSCA_A1_S` target inside `extra_labels`.
So it will generate `#define TARGET_TFM     1` inside `mbed_config.h`? and for any other target `#define TARGET_TFM     0`?

Reply:
@mikisch81 yup, everything you said is correct and the way you have written it will work just fine! I was commenting because there was a previous PR that helped standardize this kind of `#if` check in the code base (https://github.com/ARMmbed/mbed-os/pull/9163). If you accept my suggested change, it would bring your changes inline with that styling.

Reply:
Oh, and also this PR: https://github.com/ARMmbed/mbed-os/pull/9327/files

Reply:
@bridadan done